src/containers/Form.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EntityDataModelApi, DataApi } from 'lattice';
+
+import FormView from '../components/FormView';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			reportInfo: {
+				1: '',
+				2: '',
+				3: null,
+				4: '',
+				5: '',
+				6: '',
+				7: '',
+				8: '',
+				9: null,
+				10: '',
+				11: ''
+			},
+			consumerInfo: {
+				'13a': '',
```

Comment:
Be consistent with object keys. You mix using quotes and not. In general, you do not need to use quotes for the keys.